common/common_helper.sh

```diff
@@ -650,6 +650,48 @@ function kruize_local_patch() {
 	popd >/dev/null
 }
 
+function kruize_local_ros_patch() {
+ CRC_DIR="./autotune/manifests/crc/default-db-included-installation"
+	KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT="${CRC_DIR}/openshift/kruize-crc-openshift.yaml"
+	KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE="${CRC_DIR}/minikube/kruize-crc-minikube.yaml"
+
+	if [ ${CLUSTER_TYPE} == "minikube" ]; then
+      		if grep -q '"isROSEnabled": "false"' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}; then
```

Comment:
I would suggest waiting for PR [1549](https://github.com/kruize/autotune/pull/1549) to be reviewed and merged and  then update this accordingly